### PR TITLE
Create karate_brushless_whoop.txt

### PR DESCRIPTION
**Pull request by sugaarK**
this is my brushless whoop tune. its slider based to make easy to change to suit personal needs.

the quad i used to develop it is the meteor 85 from betaFPV but it has an HGLRC  ZEUS10 AIO which has logging
and it has the new hdzero whoop bundle

IMG_1185

### CODE REVIEW

File: presets/4.4/tune/karate/karate_brushless_whoop.txt [hunk 1]
```diff
@@ -0,0 +1,162 @@
+#$ TITLE: Karate brushless whoop
+#$ FIRMWARE_VERSION: 4.4
+#$ CATEGORY: TUNE
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: karate, 2s, race, brushless,  whoop 5 inch, 5", sugarK, KarateBrot
```
Comment (limonspb) [inline presets/4.4/tune/karate/karate_brushless_whoop.txt:5]: i guess need to remove 5" from keywords, and add 65mm or whatever size whoops are?

File: presets/4.4/tune/karate/karate_brushless_whoop.txt [hunk 2]
```diff
@@ -0,0 +1,162 @@
+#$ TITLE: Karate brushless whoop
```
Comment (limonspb) [inline presets/4.4/tune/karate/karate_brushless_whoop.txt:1]: "Karate brushless **2S** whoop"?
If this is a 2S. Or it's tested-good on 1S too?

File: presets/4.4/tune/karate/karate_brushless_whoop.txt [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    set motor_pwm_protocol = DSHOT600
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Dshot300
+    set dshot_bidir = ON
+    set motor_pwm_protocol = Dshot300
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Set 12 poles for whoop motors
+	set motor_poles = 12
```
Comment (limonspb) [inline presets/4.4/tune/karate/karate_brushless_whoop.txt:106]: plz replace tab with 4 spaces here

File: presets/4.4/tune/karate/karate_brushless_whoop.txt [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+
+#$ OPTION BEGIN (CHECKED): Vbat sag compensation
+	set vbat_sag_compensation = 100
+#$ OPTION END	
+
+#$ OPTION_GROUP BEGIN: Some popular RC Links
+
+    #$ OPTION BEGIN (UNCHECKED): Tracer/ELRS 250Hz
+        # Tracer/ELRS 250Hz
+        #$ INCLUDE: presets/4.3/rc_link/defaults.txt
```
Comment (limonspb) [inline presets/4.4/tune/karate/karate_brushless_whoop.txt:122]: the whole body of 250Hz / 500Hz options could be replaced with something like:
```
    #$ OPTION BEGIN (UNCHECKED): Tracer/ELRS 250Hz
        feature RX_SERIAL
        set serialrx_provider = CRSF
        #$ INCLUDE: presets/4.3/rc_link/generic/250hz_race.txt
    #$ OPTION END
```

File: presets/4.4/tune/karate/karate_brushless_whoop.txt [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+        set rc_smoothing_auto_factor = 25
+        set rc_smoothing_auto_factor_throttle = 25
+    #$ OPTION END
+#$ OPTION_GROUP END
+
+#$ OPTION_GROUP BEGIN: Optional rates
+
+    #$ OPTION BEGIN (UNCHECKED): sugarK's whoop rates
+        set thr_expo = 80
+		set roll_expo = 54
```
Comment (limonspb) [inline presets/4.4/tune/karate/karate_brushless_whoop.txt:155]: plz replace tabs with spaces here too in this section

**Review by limonspb — CHANGES_REQUESTED** [review]

File: presets/4.4/tune/karate/karate_brushless_whoop.txt (hunk 1 above)
Comment (sugaarK) [reply presets/4.4/tune/karate/karate_brushless_whoop.txt:5]: dam

File: presets/4.4/tune/karate/karate_brushless_whoop.txt (hunk 2 above)
Comment (sugaarK) [reply presets/4.4/tune/karate/karate_brushless_whoop.txt:1]: its good for 1s brushless too

File: presets/4.4/tune/karate/karate_brushless_whoop.txt (hunk 3 above)
Comment (sugaarK) [reply presets/4.4/tune/karate/karate_brushless_whoop.txt:106]: Da

File: presets/4.4/tune/karate/karate_brushless_whoop.txt (hunk 5 above)
Comment (sugaarK) [reply presets/4.4/tune/karate/karate_brushless_whoop.txt:155]: your spacing doest play nice with my editor.. will work out how to fix

Comment (limonspb) [reply presets/4.4/tune/karate/karate_brushless_whoop.txt:155]: > your spacing doest play nice with my editor.. will work out how to fix

in most places you got it good :)

File: presets/4.4/tune/karate/karate_brushless_whoop.txt (hunk 2 above)
Comment (sugaarK) [reply presets/4.4/tune/karate/karate_brushless_whoop.txt:1]: 1s is good

File: presets/4.4/tune/karate/karate_brushless_whoop.txt (hunk 3 above)
Comment (sugaarK) [reply presets/4.4/tune/karate/karate_brushless_whoop.txt:106]: da

File: presets/4.4/tune/karate/karate_brushless_whoop.txt (hunk 5 above)
Comment (sugaarK) [reply presets/4.4/tune/karate/karate_brushless_whoop.txt:155]: Da

File: presets/4.4/tune/karate/karate_brushless_whoop.txt [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+	set dshot_bidir = ON
+	set motor_pwm_protocol = Dshot300
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Set 12 poles for whoop motors
+	set motor_poles = 12
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): Dynamic idle for 11000kv 2s
+	set dyn_idle_min_rpm = 150
```
Comment (limonspb) [inline presets/4.4/tune/karate/karate_brushless_whoop.txt:110]: tabs :)

File: presets/4.4/tune/karate/karate_brushless_whoop.txt [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+	set dyn_idle_p_gain = 35
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Vbat sag compensation
+	set vbat_sag_compensation = 100
```
Comment (limonspb) [inline presets/4.4/tune/karate/karate_brushless_whoop.txt:115]: tab :)

File: presets/4.4/tune/karate/karate_brushless_whoop.txt [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+        feature RX_SERIAL 
+        set serialrx_provider = CRSF 
+        #$ INCLUDE: presets/4.3/rc_link/generic/500hz_race.txt
+    #$ OPTION END
+#$ OPTION_GROUP END
+
+#$ OPTION_GROUP BEGIN: Optional rates
+
+    #$ OPTION BEGIN (UNCHECKED): sugarK's whoop rates
+		set thr_expo = 80
```
Comment (limonspb) [inline presets/4.4/tune/karate/karate_brushless_whoop.txt:136]: tabs here still :)

File: presets/4.4/tune/karate/karate_brushless_whoop.txt [hunk 9]
```diff
@@ -0,0 +1,144 @@
+#$ TITLE: Karate brushless whoop
+#$ FIRMWARE_VERSION: 4.4
+#$ CATEGORY: TUNE
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: karate, 2s, race, brushless,  whoop, 65mm, sugarK, KarateBrot
```
Comment (limonspb) [inline presets/4.4/tune/karate/karate_brushless_whoop.txt:5]: extra space before "whoop" :)

Comment (limonspb) [inline presets/4.4/tune/karate/karate_brushless_whoop.txt:5]: can also add 1S in the keywords since it works for 1S

Comment (sugaarK) [reply presets/4.4/tune/karate/karate_brushless_whoop.txt]: I’m using amend as well 😂

**Comment by haslinghuis** [comment]
Please replace tabs with 4 spaces

**Review by limonspb — APPROVED** [review]

**Review by KarateBrot — APPROVED** [review]

**Comment by Mick51** [comment]
Hello, is it possible to update the profile for the meteor 85 HD 2022 version? It is in genfan two-blade, and the chassis needs to be changed. It would be good if the profile could be updated for the new version of the 85 please. Thanks in advance. I still have small vibrations that I can't remove despite this preset.

**Pull request merged**